error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1947,4 +1948,20 @@ Mono<T> after(CompletableFuture<T> future, boolean suppressCancel) {
       return Mono.fromFuture(() -> future, suppressCancel);
     }
   }
+
+  /**
+   * Prefer {@link Flux#fromStream(Supplier)} over {@link Flux#fromStream(Stream)}, as the former
+   * may defer initiation of the asynchronous computation until subscription.
+   */
+  static final class FluxFromStreamSupplier<T> {
+    @BeforeTemplate
+    Flux<T> before(Stream<T> stream) {
+      return Flux.fromStream(stream);
```

Comment:
This triggers an interaction with `FluxFromIterable`, so more changes are necessary; will apply.